web/templatetags/feedback_extras.py

```
@@ -0,0 +1,95 @@
+import re
+
+from django import template
+from django.utils.safestring import mark_safe
+
+register = template.Library()
+
+
+@register.filter
+def sub(value, arg):
```

Ruff (0.8.2)

Missing return type annotation for public function `sub`

(ANN201)